### PR TITLE
Remove unused rules

### PR DESCRIPTION
**Pull request by PetervDooren**
** Note: Your contribution is expected to meet the conventions and policies described in the [contribution guidelines](https://github.com/RoboCupAtHome/RuleBook/wiki/Guidelines:-Contributing) **

## Description
Removed rules which are not enforced in practice to clean up the rulebook and better reflect the situation at an actual tournament

Closes issue #

Changes proposed in this pull request:
- Removed restart rule: this rule is seldom used in practice.
- Removed the rule on moderation. LOC rarely arranges for this so don't need to add this rule. If we do want to provide moderation the LOC can specify it on their platform. 
- Updated the allocation of referees: In practice the main referee is a member of the TC and one or more team members are asked to assist. But the protocols in the rulebook are not used.
- clarify that the referee will press the emergency button, rather than ask the team to stop the robot.
- Allow multiple teammembers in the arena for the purpose of removing the robot: this is already done, despite the fact that it is technically not allowed.
- remove rule which states that a team must complete at least one stage II task to compete in the finals. This will never happen, and if it does something has gone terribly wrong in tuning the difficulty of challenges.
- fix minor \iterm \iaterm bug

## Other comments
If this request is marked as a draft, please describe your plans or any specific feedback you would like

### CODE REVIEW

File: general_rules/Procedure.tex [hunk 1]
```diff
@@ -72,15 +72,9 @@ \subsection{Collisions}
 	\item \textbf{\iterm{Major collisions}:} If a robot crushes into something during a test, the robot is immediately stopped.	Additional penalties may apply.
 
 	\item \textbf{\iterm{Functional touching}:} Robots are allowed to apply pressure on objects, push away furniture and, in general, interact with the environment using structural parts other than their manipulators.
-	This is known as \iaterm{functional touching}.
```
Comment (MatthijsBurgh) [inline general_rules/Procedure.tex:75]: Why is this change needed? `iaterm` is a valid command, see https://github.com/RoboCupAtHome/RuleBook/blob/5185332e20fa3fd4444a9930e4350ed2a1033290/setup/abbrevix.tex#L51

File: general_rules/Procedure.tex [hunk 2]
```diff
@@ -72,15 +72,9 @@ \subsection{Collisions}
...
+	This is known as \iterm{functional touching}.
 	However, the robot must clearly announce the collision-like interaction and kindly request not being stopped.\\
 	\textbf{Remark: } Referees can (and will) immediately stop a robot in case or suspicion of \emph{dangerous} behavior.
-
-	\item \textbf{Robot-Robot avoidance:} If two robots encounter each other, they both have to actively try to avoid the other robot.
-	\begin{enumerate}
-		\item A robot which is not going for a different route within a reasonable amount of time (e.g., \SI{30}{\second}) is removed.
-		\item A non-moving robot blocking the path of another robot for longer than a reasonable amount of time (e.g., \SI{30}{\second}) is removed.
-	\end{enumerate}
```
Comment (MatthijsBurgh) [inline general_rules/Procedure.tex:83]: I think this valid as the number of multi robot situations have been increasing

File: general_rules/Procedure.tex [hunk 3]
```diff
@@ -228,34 +196,6 @@ \subsection{Time limits}
 	\item \textbf{Show must go on:} On special cases, the referee may let the robot continue the test for demonstration purposes, but no additional points will be scored.
 \end{enumerate}
 
-
-
-\subsection{Restart}
```
Comment (MatthijsBurgh) [inline general_rules/Procedure.tex:233]: Restarts have been used in Stage 2 tests in the pasts. As these tests have a longer duration than stage 1.

File: general_rules/Procedure.tex (hunk 1 above)
Comment (PetervDooren) [reply general_rules/Procedure.tex:75]: Because with iaterm the result in pdf is "functional touching (.)"
As far as I can tell iaterm takes two arguments.

File: general_rules/Procedure.tex [hunk 4]
```diff
@@ -187,23 +172,6 @@ \subsection{Operators}
 Penalty may apply when using a custom operator.
 
 
-\subsection{Moderator}
```
Comment (MatthijsBurgh) [inline general_rules/Procedure.tex:190]: So do we eliminate the usage of any other language than English?

File: general_rules/Scenario.tex [hunk 5]
```diff
@@ -176,7 +176,6 @@ \subsection{Predefined Rooms and Locations}
 \label{rule:scenario_locations}
 
 Some tests in the RoboCup@Home league involve a \PredefinedLocation{} where people or objects can be found.
-Each \PredefinedLocation{} has an assigned \LocationClass{} (for instance, a \textit{Couch} and an \textit{Arm Chair} belong to the \textit{Seating} class).
```
Comment (MatthijsBurgh) [inline general_rules/Scenario.tex:179]: Any reason to remove this definition?

Comment (PetervDooren) [reply general_rules/Scenario.tex:179]: It has not been provided in past tournaments. There is also no mention of it in the rest of the rulebook. So this is a definition without use

Comment (MatthijsBurgh) [reply general_rules/Scenario.tex:179]: Fair

File: general_rules/Procedure.tex (hunk 2 above)
Comment (PetervDooren) [reply general_rules/Procedure.tex:83]: Where? In none of the tests do we use multiple robots at once. The only times this may happen is during the setup days (in which case the robots do not follow these rules) or in the final (where the robots belong to the same team)

File: general_rules/Procedure.tex (hunk 4 above)
Comment (PetervDooren) [reply general_rules/Procedure.tex:190]: No, that is not what this rule says. This rule is meant to have someone of the team be available to provide live commentary to the audience. Which is not/rarely organised by the LOC.

File: general_rules/Procedure.tex (hunk 2 above)
Comment (MatthijsBurgh) [reply general_rules/Procedure.tex:83]: I have seen situations in which two robots run the restaurant challenge at the same time. And multiple robots can be used in Open/Final challenges.

File: general_rules/Procedure.tex (hunk 4 above)
Comment (MatthijsBurgh) [reply general_rules/Procedure.tex:190]: Oh sorry didn't read carefully enough

File: general_rules/Procedure.tex (hunk 2 above)
Comment (PetervDooren) [reply general_rules/Procedure.tex:83]: In Open and Final challenges the robots belong to the same team. We do not need to specify how the robots should interact in that case since the team will opt for a working solution with two of their own robots.
As for the restaurant challenge, I have not seen it in 2019 or 2022. Don't know much about years before that. But it seems strange for that to be the only reason to include these rules. The restaurant also does not state that multiple robots may test simultaneously.

File: general_rules/Procedure.tex (hunk 3 above)
Comment (PetervDooren) [reply general_rules/Procedure.tex:233]: I don't have enough experience to say whether or not this rule helps for that.
Another question I had was whether or not a team may use this rule when their robot fails at the door. The rule does not specify how much time a team has to fix their robot.
I think this will make running a test slot more of a hastle while there are also multiple attempts per test slot.

Comment (MatthijsBurgh) [reply general_rules/Procedure.tex:233]: I leave this one for you to discuss with other TC members.

File: general_rules/Procedure.tex (hunk 2 above)
Comment (MatthijsBurgh) [reply general_rules/Procedure.tex:83]: I leave this one for you to discuss with other TC members.

**Review by ARTenshi — APPROVED** [review]
Changes have been discussed at the TC's slack channel and no extra feedback was provided.
